src/moi_oracle.jl

```diff
@@ -1,6 +1,6 @@
 
 """
-    MathOptLMO{OT <: MOI.Optimizer} <: LinearMinimizationOracle
```

Comment:
probably fine - can you give me a quick explanation for the change?

Reply:
it's just the name of the abstract MOI type which is AbstractOptimizer instead of just Optimizer